src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -178,10 +188,38 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   if (!wbsElement) {
     return res.status(404).json({ message: `wbs element with id #${updated.wbsElementId} not found` });
   }
+  const wp = await prisma.work_Package.findUnique({
+    where: {
+      wbsElementId: wbsElement.wbsElementId
+    },
+    include: {
+      expectedActivities: true,
+      deliverables: true
+    }
+  });
 
   if (updated.accepted && foundCR.type === CR_Type.STAGE_GATE) {
     const shouldChangeStatus = wbsElement.status !== WBS_Element_Status.COMPLETE;
 
+    if (wp) {
+      const wpExpectedActivities = wp.expectedActivities;
+      const wpDeliverables = wp.deliverables;
+
+      //checks for any unchecked expected activities, if there are any it will return an error
+      const uncheckedExpectedActivitiy = wpExpectedActivities.map(checkForUncheckedDescriptionBullets);
```

Comment:
you can use the Array.some method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some which is basically an ormap